### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by acrylic-style**
It is intended to stop people from posting duplicate issues. I don't know if it works, but let's try.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
```
Comment (oamaok) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: I assume these are used to prefill the issue fields? Not too familiar with the github template system.

Comment (acrylic-style) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: Maybe leave it empty then? I'm also not familiar with github issue template thing

Comment (oamaok) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: Let's see what happens if we merge this!

**Comment by oamaok** [comment]
image
Yeah, it seems that the first rows pre-fill some of the issue fields. If I select the bug template, you can see that the bug label is automatically applied.

**Pull request merged**